tinygrad/codegen/uops.py

```diff
@@ -59,8 +59,6 @@ def uop_alu_resolve(u:UOp) -> sint:
   elif u.uop is UOps.ALU and u.arg == BinaryOps.ADD: return uop_alu_resolve(u.vin[0]) + uop_alu_resolve(u.vin[1])
   else: raise RuntimeError(f"ALU resolve fail @ {u.uop}")
 
-def phi_resolve_acc(u:UOp) -> UOp: return u if u.uop is UOps.DEFINE_ACC else phi_resolve_acc(u.vin[0])
```

Comment:
the intention behind this was to group TC+upcast opts as well. Their ACC is nested into the PHI.
`test.test_linearizer.TestLinearizerOpts.test_tensor_core_opts`
Your branch (r_2_4_2_2_2_4_2_8_8_2_4_2_4n2)
```
*((device half2*)(data0+alu7)) = half2(acc0.x,acc0.y)
```

master:
```
*((device half2*)(data0+alu7)) = acc0;
```

Reply:
thanks, was looking for this case. did I miss anything else?

Reply:
confirmed with the process replay tests this covers all the cases now.

Reply:
thanks